### PR TITLE
Edit addp and addi

### PR DESCRIPTION
**Pull request by tanxuanyun**

### CODE REVIEW

**Comment by chengda300** [comment]
Looks good though actually phone number seems to be needed now

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,28 +66,29 @@ Format: `help`
 
 Adds a person to InterNUS.
 
-Format: `addp n/NAME e/EMAIL [p/PHONE_NUMBER] [i/INTERNSHIP] [t/TAG]…​`
+Format: `addp n/NAME e/EMAIL [p/PHONE_NUMBER] [t/TAG]…​ [l/LINK_INDEX]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0). Phone number is optional, and internship is optional too.
+A person can have any number of tags (including 0). Phone number is optional, and link index is optional too.
 </div>
```
Comment (ish1506) [inline docs/UserGuide.md:73]: Maybe we should explain what the `LINK_INDEX` is.
```suggestion
- The link index refers to the index number shown in the internship list.
</div>
```

Comment (ish1506) [reply docs/UserGuide.md:73]: Likewise for `addi`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -66,28 +66,29 @@ Format: `help`
 
 Adds a person to InterNUS.
 
-Format: `addp n/NAME e/EMAIL [p/PHONE_NUMBER] [i/INTERNSHIP] [t/TAG]…​`
+Format: `addp n/NAME e/EMAIL [p/PHONE_NUMBER] [t/TAG]…​ [l/LINK_INDEX]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0). Phone number is optional, and internship is optional too.
+A person can have any number of tags (including 0). Phone number is optional, and link index is optional too.
```
Comment (ish1506) [inline docs/UserGuide.md:72]:
```suggestion
A person can have any number of tags (including 0). Link index is optional.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -66,28 +66,29 @@ Format: `help`
...
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Date of interview can be left blank, since it is possible that an interview is not scheduled yet.
+Link index is optional too.
 </div>
 
 Examples:
-* `addi n/TikTok r/Data Engineer s/rejected`
+* `addi n/TikTok r/Data Engineer s/rejected l/1`
 * `addi n/Grab r/Full Stack Developer s/awaiting interview d/2020-12-20`
```
Comment (ish1506) [inline docs/UserGuide.md:92]:
```suggestion
* `addi c/Grab r/Full Stack Developer s/PENDING d/2020-12-20 12:00`
```
The input formats here are incorrect.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -66,28 +66,29 @@ Format: `help`
 
 Adds a person to InterNUS.
 
-Format: `addp n/NAME e/EMAIL [p/PHONE_NUMBER] [i/INTERNSHIP] [t/TAG]…​`
+Format: `addp n/NAME e/EMAIL [p/PHONE_NUMBER] [t/TAG]…​ [l/LINK_INDEX]`
```
Comment (ish1506) [inline docs/UserGuide.md:69]:
```suggestion
Format: `addp n/NAME e/EMAIL p/PHONE_NUMBER [t/TAG]…​ [l/LINK_INDEX]`
```
Phone number is still compulsory in v1.2

**Review by ish1506 — COMMENTED** [review]
Hi, I left some comments regarding bugs.

**Pull request merged**